README.md

```
@@ -1,52 +1,98 @@
 <div align="center">
 
-<img src="assets/logo.svg" alt="VEET — Universal Linux App Uninstaller" width="620"/>
+<img src="assets/logo.svg" alt="VEET — Universal Linux App Uninstaller" width="600"/>
 
-<br/><br/>
+<br/>
 
+[![CI](https://github.com/swadhinbiswas/veet/actions/workflows/ci.yml/badge.svg)](https://github.com/swadhinbiswas/veet/actions/workflows/ci.yml)
 [![Go Version](https://img.shields.io/badge/Go-1.22+-00ADD8?style=flat-square&logo=go)](https://go.dev)
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg?style=flat-square)](LICENSE)
 [![Platform](https://img.shields.io/badge/Platform-Linux-FCC624?style=flat-square&logo=linux&logoColor=black)](#)
```

markdownlint-cli2 (0.23.2)

[warning] No empty links

(MD042, no-empty-links)